src/components/services/Radarr.vue

```diff
@@ -77,6 +77,17 @@ export default {
           }
         })
         .catch(handleError);
+      this.fetch(`${this.apiPath}/queue/details?apikey=${this.item.apikey}`)
```

Comment:
Just to be sure, is the `/queue/details` endpoint exists on the old API ?

Reply:
I've no idea where to find documentation on the old API. Should I just wrap this new section in `if (!this.item.legacyApi)` in case?

Reply:
I couldn't find it either :( 
I don't even know if the legacy API is used anymore. Let's put the `if` as you suggest for now if that's ok for you :+1:

Reply:
I have done this! Cheers.